File: algorithms/2_medium/M/00453_minimum_moves_to_equal_array_elements/solution.cpp
class Solution {
public:
    static int minMoves(const vector<int>& nums) {
        const int min{ *min_element(nums.cbegin(), nums.cend()) };
        int count{ 0 };

        for (const auto& num : nums) {
            count += num - min;
        }

        return count;
    }
};

File: algorithms/2_medium/M/00453_minimum_moves_to_equal_array_elements/solution.rs
impl Solution {
    pub fn min_moves(nums: Vec<i32>) -> i32 {
        let min = *nums.iter().min().unwrap();
        let mut count = 0_i32;

        for num in nums.into_iter() {
            count += num - min;
        }

        count
    }
}
